README.md

```diff
@@ -74,3 +74,35 @@ To try out Rellic you can do the following, given a LLVM bitcode file of your ch
 ```shell
 ./rellic-build/rellic-decomp --input mybitcode.bc --output /dev/stdout
 ```
+
+### Docker image
+
+The Docker image should provide an environment which can set-up, build, and run rellic.
+
+To build the docker image:
+```sh
+docker build -t rellic-decomp-40 --build-arg LLVM_VERSION=4.0 .
+```
+
+To run the decompiler, the entrypoint has already been set, but make sure the bitcode you are decompiling is the same as LLVM version as the decompiler, and run:
+
+```sh
+docker run --rm -t -i \
+  -v $(pwd):/test -w /test \
+  -u $(id -u):$(id -g) \
+  rellic-decomp-40 --input ./tests/tools/decomp/issue_4.bc --output /dev/stdout
```

Comment:
The `tests/tools/decomp/` folder contains C sources. The compilation of `issue_4.bc` is missing here.

Reply:
Maybe also move the `docker-decomp-entry-point.sh` to `scripts/` if possible. Just to keep things neat. Unless there's some convention of such scripts being in the root of the repo. I really don't have too much experience with setting up docker images.

Reply:
I added a command to build the bc file in the README.

Also moved `docker-decomp-entry-point.sh` to `scripts/`